localization/languages/en-US.json

```diff
@@ -137,6 +137,11 @@
     "settingsBlockedRequestCount": {
       "unsafeHTML": "So far, Min has blocked <strong></strong> ads and trackers."
     },
+    "settingsCustomBangs": "Custom bangs",
+    "settingsCustomBangsAdd": "Add custom bang", 
+    "settingsCustomBangsPhrase": "Phrase",
+    "settingsCustomBangsSnippet": "Snippet",
+    "settingsCustomBangsRedirect": "Redirect",
```

Comment:
These descriptions may not be clear to someone who isn't already familiar with how bangs work; I would suggest "Phrase", "Description (optional)", "URL"

Reply:
Well right now it isn't optional, it's required, but I'll do that.